### PR TITLE
Added temperature and humidity to packet

### PR DESCRIPTION
**Pull request by Ilias-Tsigkogiannis**
Added temperature and humidity to packet, in order to make sample compatible with our advanced tutorials at docs.microsoft.com

### CODE REVIEW

File: device/samples/DeviceClientAmqpSample/Program.cs [hunk 1]
```diff
@@ -15,6 +15,10 @@ class Program
         private const string DeviceConnectionString = "<replace>";
 
         private static int MESSAGE_COUNT = 5;
+        private static String deviceId = "MyCSharpDevice";
```
Comment (tameraw) [inline device/samples/DeviceClientAmqpSample/Program.cs]: >deviceId [](start = 30, length = 8)

DeviceId is provided in the connection string, so no need for this variable. Here and in the other samples.

File: device/samples/DeviceClientAmqpSample/Program.cs [hunk 2]
```diff
@@ -48,8 +52,11 @@ static void Main(string[] args)
 
             for (int count = 0; count < MESSAGE_COUNT; count++)
             {
-                dataBuffer = Guid.NewGuid().ToString();
+                temperature = rnd.Next(20, 30);
+                humidity = rnd.Next(60, 80);
+                dataBuffer = string.Format("{{\"deviceId\":\"{0}\",\"messageId\":{1},\"temperature\":{2},\"humidity\":{3}}}", deviceId, count, temperature, humidity);
                 Message eventMessage = new Message(Encoding.UTF8.GetBytes(dataBuffer));
+                eventMessage.Properties.Add("temperatureAlert", (temperature > 28) ? "true" : "false");
```
Comment (tameraw) [inline device/samples/DeviceClientAmqpSample/Program.cs]: >28 [](start = 79, length = 2)

Please move to a private const.

File: device/samples/DeviceClientAmqpSample/Program.cs (hunk 1 above)
Comment (Ilias-Tsigkogiannis) [inline device/samples/DeviceClientAmqpSample/Program.cs]: we use the deviceId in the SDK samples for all languages, (C, Python, Node), as well as in the Happy Path.

**Comment by tameraw** [comment]
:shipit:

**Comment by tameraw** [comment]
Merged thru the gate.

**Pull request closed without merging**